src/main/java/com/bisnode/opa/client/rest/JsonBodyPublisher.java

```diff
@@ -0,0 +1,36 @@
+package com.bisnode.opa.client.rest;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AccessLevel;
+import lombok.SneakyThrows;
+import lombok.experimental.FieldDefaults;
+
+import java.net.http.HttpRequest;
+import java.nio.ByteBuffer;
+import java.util.concurrent.Flow;
+
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+public class JsonBodyPublisher implements HttpRequest.BodyPublisher {
+
+    HttpRequest.BodyPublisher stringBodyPublisher;
+
+    public static HttpRequest.BodyPublisher of(Object body) {
+        return new JsonBodyPublisher(body, new ObjectMapper());
```

Comment:
`ObjectMapper` is heavy in creation. If possible, try to create it once and reuse.